app/src/main/java/com/infomaniak/swisstransfer/ui/theme/Theme.kt

```diff
@@ -47,10 +52,11 @@ fun SwissTransferTheme(
 }
 
 @Composable
-fun isDarkTheme(): Boolean {
-    // rememberMutableStateOf
-    // TODO check in realm. If system, isSystemDark, otherwise,
-    return isSystemInDarkTheme()
+fun isDarkTheme(settingsViewModel: SettingsViewModel): Boolean {
+    val appSettings by settingsViewModel.appSettingsFlow.collectAsStateWithLifecycle(null)
+    return appSettings?.let {
+        if (it.theme == Theme.SYSTEM) isSystemInDarkTheme() else it.theme == Theme.DARK
+    } ?: isSystemInDarkTheme()
```

Comment:
Is `System` in default from kmp?

Reply:
Yes it is.

<img width="555" alt="Capture d’écran 2024-09-19 à 09 28 53" src="https://github.com/user-attachments/assets/313d634b-aeba-49c4-bc1d-f00847b98cda">